minhash_service/minhash_service/minhash/io.py

```diff
@@ -138,6 +138,18 @@ def remove_signature(sample_id: str) -> bool:
     return False
 
 
+def check_signature(sample_id: str) -> bool:
+    """Check if signature exist and has been added to the index."""
+    LOG.info("Checking signature file: %s", signature_file)
+    # check that signature exist
+    try:
+        signature_file = get_signature_path(sample_id, check=True)
```

Comment:
Hmm. Thinking about the exception here. Would it not be more straightforward to have a `check_signature_path_exists(sample_id) -> bool` ID? Or just return `None` if not present. So that you could:

```
signature_file = get_signature_path(sample_id)
return signature_file is not None
```